### PR TITLE
Bypass Email to clients

### PR DESCRIPTION
**Pull request by jSylvestre**
image

### CODE REVIEW

**Comment by jSylvestre** [comment]
image

File: Anlab.Mvc/Views/Lab/Confirmation.cshtml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 }
 <div class="col">
     @Html.Partial("_OrderDetails")
-    <form asp-controller="Lab" asp-action="Confirmation" asp-route-id="@Model.Order.Id" method="post">
+    <form id="form" asp-controller="Lab" asp-action="Confirmation" asp-route-id="@Model.Order.Id" method="post">
```
Comment (srkirkland) [inline Anlab.Mvc/Views/Lab/Confirmation.cshtml]: @jSylvestre this switch method is ok but it's also easy in the future if you just make 2 forms, then no javascript needed

Comment (jSylvestre) [reply Anlab.Mvc/Views/Lab/Confirmation.cshtml]: @srkirkland There is form data there. If there were 2 forms would that data be included in both?

Comment (srkirkland) [reply Anlab.Mvc/Views/Lab/Confirmation.cshtml]: @jSylvestre yea

Comment (srkirkland) [reply Anlab.Mvc/Views/Lab/Confirmation.cshtml]: @jSylvestre maybe not the best option for this specific situation

**Comment by jSylvestre** [comment]
image

**Comment by srkirkland** [comment]
@jSylvestre looks good 🚢

**Pull request merged**